src/NUnitFramework/Directory.Build.props

```diff
@@ -25,7 +25,7 @@
 
   <PropertyGroup Condition="'$(BuildingInsideVisualStudio)' != 'true'">
     <!-- Ideally this is always enabled, but that tends to hurt developer productivity -->
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
No.

Reply:
Already reverted, accidentally commited it while testing locally